src/test/java/marquez/api/mappers/NamespaceResponseMapperTest.java

```diff
@@ -44,7 +44,8 @@ public void testSingleNamespace() {
     assertThat(Timestamp.valueOf(mappedResponse.getCreatedAt()))
         .isEqualTo(descriptionNamespace.getCreatedAt());
     assertThat(mappedResponse.getOwner()).isEqualTo(descriptionNamespace.getOwnerName());
-    assertThat(mappedResponse.getDescription().orElse(null)).isEqualTo(descriptionNamespace.getDescription());
+    assertThat(mappedResponse.getDescription().orElse(null))
```

Comment:
Since we know that description here is not null, shouldn't we add a test for mapperResponse.getDescription.isPresent() ?